nextflow.config

```diff
@@ -28,125 +39,256 @@ params {
-        process.debug          = true
-        cleanup                = false
-        nextflow.enable.configProcessNamesValidation = true
+    podman {
+        podman.enabled          = true
+        conda.enabled           = false
+        docker.enabled          = false
+        singularity.enabled     = false
+        shifter.enabled         = false
+        charliecloud.enabled    = false
+        apptainer.enabled       = false
+    }
+    shifter {
+        shifter.enabled         = true
+        conda.enabled           = false
+        docker.enabled          = false
+        singularity.enabled     = false
+        podman.enabled          = false
+        charliecloud.enabled    = false
+        apptainer.enabled       = false
+    }
+    charliecloud {
+        charliecloud.enabled    = true
+        conda.enabled           = false
+        docker.enabled          = false
+        singularity.enabled     = false
+        podman.enabled          = false
+        shifter.enabled         = false
+        apptainer.enabled       = false
+    }
+    apptainer {
+        apptainer.enabled       = true
+        apptainer.autoMounts    = true
+        conda.enabled           = false
+        docker.enabled          = false
+        singularity.enabled     = false
+        podman.enabled          = false
+        shifter.enabled         = false
+        charliecloud.enabled    = false
+    }
+    wave {
+        apptainer.ociAutoPull   = true
+        singularity.ociAutoPull = true
+        wave.enabled            = true
+        wave.freeze             = true
+        wave.strategy           = 'conda,container'
+    }
+    gitpod {
+        executor.name           = 'local'
+        executor.cpus           = 4
+        executor.memory         = 8.GB
     }
+    test         { includeConfig "conf/test.config"       }
     crg          { includeConfig "conf/crg.config"        }
     crg_slurm    { includeConfig "conf/crg_slurm.config"  }
-    test         { includeConfig "conf/test.config"       }
     test_learn   { includeConfig "conf/test_learn.config" }
     test_stub    { includeConfig "conf/test_stub.config"  }
     local        { includeConfig "conf/local.config"      }
+    test_full    { includeConfig 'conf/test_full.config'  }
 }
 
+// Load nf-core custom profiles from different Institutions
+includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/nfcore_custom.config" : "/dev/null"
 
-// Nextflow plugins
-plugins {
-    id 'nf-schema@2.0.0' // Validation of pipeline parameters and creation of an input channel from a sample sheet
+// Load nf-core/deepmodeloptim custom profiles from different institutions.
+// TODO nf-core: Optionally, you can add a pipeline-specific nf-core config at https://github.com/nf-core/configs
+// includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/pipeline/deepmodeloptim.config" : "/dev/null"
```

Comment:
This line can be uncommented once you move the pipeline to nf-core. A file `deepmodeloptim.config` will be automatically created at the nf-core/configs repo. This is to allow having a pipeline specific config.